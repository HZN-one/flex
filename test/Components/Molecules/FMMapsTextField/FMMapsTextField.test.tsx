/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

import { FMMapsTextField } from "@Molecules";

describe("FMMapsTextField", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <FMMapsTextField
        testID="maps-textfield-test"
        label="Label test"
        buttonLabel="Label Test"
      />
    );
    expect(container).toMatchSnapshot();
  });
});
