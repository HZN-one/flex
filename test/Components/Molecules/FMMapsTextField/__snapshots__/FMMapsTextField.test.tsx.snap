// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FMMapsTextField should match snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-0"
    data-testid="maps-textfield-test"
  >
    <div
      class="MuiBox-root css-1itv5e3"
    >
      <span
        class="MuiTypography-root MuiTypography-captionSemiBold css-37jdci-MuiTypography-root"
        data-testid="typography-maps-label"
      >
        Label test
         
      </span>
    </div>
    <div
      class="MuiBox-root css-79elbk"
    >
      <div
        aria-hidden="true"
        class="MuiBackdrop-root css-1kqc2ut-MuiBackdrop-root"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  css-vn0cak-MuiButtonBase-root-MuiButton-root"
          data-testid="button-input-maps"
          tabindex="0"
          type="button"
        >
          Label Test
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
      <img
        class="MuiBox-root css-s22gmu"
        src="https://storage.googleapis.com/dev-hzn-assets/icons/maps-placeholder.webp"
      />
    </div>
    <div
      class="MuiBox-root css-1hw29i9"
    >
      <div
        class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root flex-textfield-no-label css-wb57ya-MuiFormControl-root-MuiTextField-root"
      >
        <div
          class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-multiline css-68u1dt-MuiInputBase-root-MuiInput-root"
        >
          <textarea
            aria-invalid="false"
            class="MuiInput-input MuiInputBase-input MuiInputBase-inputMultiline css-66dh3a-MuiInputBase-input-MuiInput-input"
            id="mui-1"
            style="height: 0px; overflow: hidden;"
          />
          <textarea
            aria-hidden="true"
            class="MuiInput-input MuiInputBase-input MuiInputBase-inputMultiline css-66dh3a-MuiInputBase-input-MuiInput-input"
            readonly=""
            style="visibility: hidden; position: absolute; overflow: hidden; height: 0px; top: 0px; left: 0px; transform: translateZ(0); padding: 0px; width: 100%;"
            tabindex="-1"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
